contracts/SnapshotToken/SnapshotToken.sol

```diff
@@ -1,15 +1,18 @@
 pragma solidity 0.4.15;
 
 import '../Snapshot/DailyAndSnapshotable.sol';
-import '../Standards/IERC667Token.sol';
-import '../Standards/IERC667Callback.sol';
+import '../Standards/IERC223Token.sol';
+import '../Standards/IERC223Callback.sol';
+import '../Standards/IERC677Token.sol';
+import '../Standards/IERC677Callback.sol';
```

Comment:
Do we still plan to support both? Didn't we decide to keep ERC223?

Reply:
With this PR it will support both. We don't have to use both though. The overhead for supporting ER677 is low. (Unless you want to drop the entire Allowance mechanis and go pure ERC223, but than we loose ERC20 support :) )

Reply:
OK agree